site/content/docs/5.2/forms/checks-radios.md

```diff
@@ -49,7 +49,13 @@ Checkboxes can utilize the `:indeterminate` pseudo class when manually set via J
 
 Add the `disabled` attribute and the associated `<label>`s are automatically styled to match with a lighter color to help indicate the input's state.
 
-{{< example >}}
+{{< example class="bd-example-indeterminate">}}
```

Comment:
Missing `stackblitz_add_js="true"` here

Reply:
Added but I think that they haven't landed in Boosted yet !

Reply:
Oh my bad. I'll revert it for consistency.